src/@types/index.ts

```diff
@@ -0,0 +1 @@
+export type EventHandlerType = any;
```

Comment:
What's this for?

Reply:
So, for the `ProfileHeaderLogo` there's an `onLoad?` prop, I was having trouble getting the type to work since if an element is passed it's not required to have `onLoad`. It's utilized in the `ProfileHeaderMenuItem` for `onClick` also.